perses/annihilation/relative.py

```diff
@@ -2629,3 +2629,2206 @@ def _handle_exceptions(self):
+        "is_unique_new = step(is_unique_new1 + is_unique_new2 - 0.1);",
+        "w_scale = {w_scale};"
+    ]
+
+    # electrostatic exception
+    _default_exception_expression_list = [
+
+        "(U_electrostatics + U_sterics) * rest_scale;",
+
+        # Define rest scale
+        "rest_scale = is_rest * lambda_rest_scale * lambda_rest_scale + is_inter * lambda_rest_scale + is_nonrest;",
+
+        # Define electrostatics functional form
+        f"U_electrostatics = {ONE_4PI_EPS0} * chargeProd * erfc(alpha * r)/ r_eff_electrostatics;",
+
+        # Define sterics functional form
+        "U_sterics = 4 * epsilon * x * (x - 1.0);"
+        "x = (sigma / r_eff_sterics)^6;"
+
+        # Define chargeProd (with alchemical scaling)
+        "chargeProd = is_unique_old * old_charge_scalar + is_unique_new * new_charge_scalar + is_core * (old_charge_scalar + new_charge_scalar) + is_environment * chargeProd_old;",
+        "old_charge_scalar = lambda_electrostatics_old * chargeProd_old;",
+        "new_charge_scalar = lambda_electrostatics_new * chargeProd_new;",
+
+        # Define sigma (with alchemical scaling)
+        "sigma = is_unique_old * old_sigma_scalar + is_unique_new * new_sigma_scalar + is_core * (old_sigma_scalar + new_sigma_scalar) + is_environment * sigma_old;",
+        "old_sigma_scalar = lambda_sterics_old * sigma_old;",
+        "new_sigma_scalar = lambda_sterics_new * sigma_new;",
+
+        # Define epsilon (with alchemical scaling)
+        "epsilon = is_unique_old * old_epsilon_scalar + is_unique_new * new_epsilon_scalar + is_core * (old_epsilon_scalar + new_epsilon_scalar) + is_environment * epsilon_old;",
+        "old_epsilon_scalar = lambda_sterics_old * epsilon_old;",
+        "new_epsilon_scalar = lambda_sterics_new * epsilon_new;",
+
+        # Define alpha
+        "alpha = sqrt(-log(2 * delta) / r_cutoff);",
+        "delta = {delta};",
+        "r_cutoff = {r_cutoff};",
+
+        # Define r_eff
+        "r_eff_electrostatics = sqrt(r^2 + w_electrostatics^2);",
+        "r_eff_sterics = sqrt(r^2 + w_sterics^2);",
+
+        # Define 4th dimension terms:
+        "w_electrostatics = is_unique_old * lambda_electrostatics_new * w_scale + is_unique_new * lambda_electrostatics_new * w_scale;",
+        # we want w for unique old atoms to go from 0 to 1
+        "w_sterics = is_unique_old * lambda_sterics_new * w_scale + is_unique_new * lambda_sterics_old * w_scale;",
+        "w_scale = {w_scale};"
+
+    ]
+
+    _default_nonbonded_expression = ' '.join(_default_nonbonded_expression_list)
+    _default_exception_expression = ' '.join(_default_exception_expression_list)
+
+    def __init__(self,
+                 topology_proposal,
+                 current_positions,
+                 new_positions,
+
+                 # scaling arguments
+                 scale_regions=None,
+
+                 r_cutoff=1.4 * unit.nanometers,  # default cutoff for reaction field electrostatics
```

Comment:
change this to default to None
In the init, set it to whatever the og system nb cutoff is. If the og system's nb method is NoCutoff, set r_cutoff to 100

Reply:
fixed